benchmark/source_map/prepare-stack-trace.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const common = require('../common.js');
+const assert = require('assert');
+
+const options = {
+  flags: ['--expose-internals'],
+};
+
+const bench = common.createBenchmark(
+  main,
+  {
+    operation: ['non-node-error'],
+    n: [1e5],
+  },
+  options,
+);
+
+function ErrorGetCallSites() {
```

Comment:
@legendecas is there a better way to create this list of CallSite objects? Found that on https://github.com/nodejs/node/blob/main/benchmark/util/get-callsite.js#L12

Reply:
Oh, sorry! I misunderstood that it was just in the error case, I will remove the file! Thanks